jobs/email-reminder/email_reminder.py

```diff
@@ -19,16 +19,17 @@
 
 import requests
 import sentry_sdk  # noqa: I001, E501; pylint: disable=ungrouped-imports; conflicts with Flake8
-from flask import Flask
-from legal_api.models import Business, Filing, db  # noqa: I001
+from flask import Flask, current_app
+from legal_api.models import Filing, LegalEntity, db  # noqa: I001
```

Comment:
this should move to use the business-model
https://github.com/bcgov/lear/tree/dev_legal_name_changes/python/common/business-registry-model

Reply:
@thorwolpert what do you want to do about this here? the email reminder job references some of the service classes in legal api - grabbing auth token & some feature flag stuff

Do you want us to cut another ticket to create a common module for services and come back and integrate? or just stick with importing all of the entire legal api module?

Reply:
Approving PR to reference legal-api as a dependency @thorwolpert.

If you want any changes with regards to my previous comment, let me know.  I can cut a new ticket to address it.